buildSrc/src/main/java/org/elasticsearch/gradle/JdkDownloadPlugin.java

```diff
@@ -48,13 +48,14 @@
 public class JdkDownloadPlugin implements Plugin<Project> {
 
     private static final String REPO_NAME_PREFIX = "jdk_repo_";
+    private static final String CONTAINER_NAME = "jdks";
 
     @Override
     public void apply(Project project) {
         NamedDomainObjectContainer<Jdk> jdksContainer = project.container(Jdk.class, name ->
             new Jdk(name, project)
         );
-        project.getExtensions().add("jdks", jdksContainer);
+        project.getExtensions().add(CONTAINER_NAME, jdksContainer);
```

Comment:
👍 Yay! I have a note to go through all our stuff and replace the million duplicated and hard-coded strings for things like task and extension names. We should make it habit to replace these occurrences as we come across them.